root/etc/udev/rules.d/98-nmprepare.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="net", KERNEL!="lo", RUN+="/usr/bin/nm-prepare $name"
```

Comment:
This can go as well as the rule is already there (in d4cb4ea7f4f84396542ebd1e9c905e032e11cedd).